### PR TITLE
[T7][T09-C2]

### PR DESCRIPTION
**Pull request by cheec**
V0.0 and V0.1:
- Implement basic CRUD for tasks
- Update various UI parts
- Refactor most of the code, AddressBook -> Jimi

### CODE REVIEW

**Comment by cheec** [comment]
@ShadowLoner17 @Syltaris @yunxz

File: README.md [hunk 1]
```diff
@@ -1,34 +1,33 @@
...
+
+In our daily lives, we often find ourselves striving to be as successful as we can in whatever we do - be it in our academic pursuits or in our career choices. So naturally, this begs the question, how does one achieve success? Quite a tough question that many ask themselves. We can, however, attempt to answer a question like this by drawing parallels between our lives and the lives of successful individuals. Interestingly enough, all these success stories tend to share a common theme. They all seem to gravitate towards one key secret to success: **to always have a plan, be on track and be organized.**
+
+This provided the impetus for our team to develop a task managing app. Now, you're probably thinking to yourself, "Why would I need another task manager like this when I can pick from thousands of other competing products, in the process, entirely destroying your business model and extinguishing any pathetic desires you have of ever making it in the field of software engineering?" 
+<br><br><br>
+Well, you're right... you don't. **NOT UNLESS YOU'RE SOMEONE LIKE** [**JIM.**](http://www.comp.nus.edu.sg/~cs2103/AY1617S1/contents/handbook.html#handbook-project-product) <br><br>
```
Comment (nirandiw) [inline README.md]: Can you improve the mock up GUI a bit more so I can see how each task is shown and what details of a task is visible. i.e. at time t what would Jim's task manager look like

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -269,65 +269,311 @@ b. Require developers to download those libraries manually (this creates extra w
...
+
+1a. The input new save directory is invalid.
+
+>1a1. App shows error message to user.
+> Use case ends.
+
+1b. The input new save directory is the same as the original save directory.
+
+>1b1. App shows error message to user.
+> Use case ends.
```
Comment (nirandiw) [inline docs/DeveloperGuide.md]: nice set of use-cases

File: docs/UserGuide.md [hunk 3]
```diff
@@ -1,134 +1,227 @@
...
+* `add "Have school camp" on 10 October to 18 October, 10am to 5pm`
 
-> Deletes the person at the specified `INDEX`. 
-  The index refers to the index number shown in the most recent listing.<br>
-  The index **must be a positive integer** 1, 2, 3, ...
+<br><br>
+
+#### <a id="com"></a>Complete a task: `complete`
+Mark a task as completed. 
+Format: `complete TASK_INDEX`
```
Comment (nirandiw) [inline docs/UserGuide.md]: From where can the user get this index? Can't see it in your mock GUI

File: docs/UserGuide.md [hunk 4]
```diff
@@ -1,134 +1,227 @@
...
+
+>     * `/n` : name of task
+>     * `/sd` and `/ed`: start date and end date
+>     * `/st` and `/et` : start time and end time <br><br>
+> * If you want to undo your edit, use the [`undo`](#undo) command.
+> * The index refers to the index number shown in the most recent listing.
+> * The index **must be a positive integer** 1, 2, 3.
+
+Examples:
+* `edit 2 /n buy food`
```
Comment (nirandiw) [inline docs/UserGuide.md]: Can I edit multiple attributes at once?

File: docs/UserGuide.md [hunk 5]
```diff
@@ -1,134 +1,227 @@
...
+
+#### <a id="undo"></a>Undo previous action: `undo`
+Undos the previous action done in the task manager. <br>
+Format: `undo`
+
+<br><br>
+
+#### <a id="find"></a>Find all tasks relevant to keywords you input: `find`
+Finds tasks which details contain any of the given keywords.<br>
+Format: `find KEYWORD [MORE_KEYWORDS]`
```
Comment (nirandiw) [inline docs/UserGuide.md]: Can you show completed tasks if i want to view it? Consider it.

**Comment by nirandiw** [comment]
Hi guys, 
I have added some comments. Do take note of it. Furthermore, if you guys finish these commands consider adding priority, tags. I think your team has the potential to do more.

**Pull request closed without merging**